board.py

```diff
@@ -0,0 +1,84 @@
+from config import KEY, TOKEN, USERNAME
+import requests
+from todo_item import todo_item
+
+class board:
+
+    def __init__(self):
+        """
+        Initialisation
+        Check whether board exists. If board exists then obtain the board and list IDs.
+
+        If board does not exist then create board and repeat loop to obtain IDs.
+        """
+        BOARD_NAME = 'MyTodos'
+        self.board_id = 0
+        while self.board_id == 0:
+            boards = requests.get(f'https://api.trello.com/1/members/{USERNAME}/boards?key={KEY}&token={TOKEN}').json()
+            for board in boards:
+                if board['name'] == BOARD_NAME:
+                    self.board_id = board['id']
+                    lists = requests.get(f'https://api.trello.com/1/boards/{self.board_id}/lists?key={KEY}&token={TOKEN}').json()
+                    for list in lists:
+                        if list['name'] == 'To Do':
+                            self.todo_list_id = list['id']
+                        elif list['name'] == 'Done':
+                            self.done_list_id = list['id']
+            if self.board_id == 0:
+                print(f'Creating new board')
+                requests.post(f'https://api.trello.com/1/boards/?key={KEY}&token={TOKEN}&name={BOARD_NAME}')
```

Comment:
You can use the `params` argument to avoid having to build up the query string manually here.